diff --git a/node_modules/@effection/node/src/child_process.ts b/node_modules/@effection/node/src/child_process.ts
index 3ce078d..e31a9a9 100644
--- a/node_modules/@effection/node/src/child_process.ts
+++ b/node_modules/@effection/node/src/child_process.ts
@@ -35,7 +35,6 @@ function *supervise(child: ChildProcess, command: string, args: readonly string[
 
 export function *spawn(command: string, args?: ReadonlyArray<string>, options?: SpawnOptions): Operation {
   let child = childProcess.spawn(command, args || [], Object.assign({}, options, {
-    shell: true,
     detached: true,
   }));
   return yield resource(child, supervise(child, command, args));
