diff --git a/node_modules/@effection/node/dist/child_process.js b/node_modules/@effection/node/dist/child_process.js
index 83ab35e..a0f6e41 100644
--- a/node_modules/@effection/node/dist/child_process.js
+++ b/node_modules/@effection/node/dist/child_process.js
@@ -32,7 +32,7 @@ function* supervise(child) {
 }
 function* spawn(command, args, options) {
     let child = childProcess.spawn(command, args, Object.assign({}, options, {
-        shell: true,
+        shell: !options.shell ? true : options.shell,
         detached: true,
     }));
     return yield effection_1.resource(child, supervise(child));
